docs/logs/getting-started/README.md

```diff
@@ -61,6 +61,16 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+Adding logging providers, obtaining `ILogger` instance, etc. might be done
+differently based on the type of the application. Check the respective
```

Comment:
"check the respective documents" seems to indicate that this doc promises to provide/maintain a list of such documents, which is not the case?